src/client/components/app-header/app-header.vue

```diff
@@ -4,21 +4,30 @@
       <h1 class="md-title">Keukentafeltool</h1>
     </nuxt-link>
     <div class="md-toolbar-section-end">
-      <md-button
-        class="md-icon-button"
-        @click="$emit('refresh')"
-      >
-        <md-icon>refresh</md-icon>
-      </md-button>
+      <refresh-button
+        :active="hasFeatures"
```

Comment:
In HTML buttons already have an `active` state which is different from what you want to use it for. You want to use the opposite of `disabled`, which in HTML/CSS is `enabled`.